consensus/parlia/snapshot.go

```diff
@@ -43,6 +44,7 @@ type Snapshot struct {
 
 	Number           uint64                            `json:"number"`                // Block number where the snapshot was created
 	Hash             common.Hash                       `json:"hash"`                  // Block hash where the snapshot was created
+	TurnTerm         uint64                            `json:"turn_term"`             // Term of `turn`, meaning the consecutive number of blocks a validator receives priority for block production
```

Comment:
I will suggest run a fullnode to use this version of code to sync from genesis to latest code on testnet or mainnet, to prove that it is fully compatible with history blocks.

Reply:
> I will suggest run a fullnode to use this version of code to sync from genesis to latest code on testnet or mainnet, to prove that it is fully compatible with history blocks.

accepted

Reply:
a node is running fullsync againse chapel net, almost catch up the newest block